packages/common/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-v0.4.0
+## 2.0.0
+
+### Major Changes
+
+- Added version locking to adaptors that uses `openfn/language-common`
+  dependency to ensure consistent and predictable behavior across different
+  environments.
+- Removed `http` on `common` to improve compatibility and reduce potential
```

Comment:
Again we don't need to justify the change here - although as it's a breaking change it's useful to mention the workaround,

I think this diff should be something like `removed the http namespace. Adaptors should use the new request function`.

Oh, and the new request helper should be mentioned in this change!